docs/.custom_wordlist.txt

```diff
@@ -0,0 +1,26 @@
+Permalink
+reST
+RST
+reStructuredText
+UI
+favicon
+installable
+Makefile
+yaml
+RTD
+readthedocs
+MicroOVN
+ovsdb
+OVN
+northd
```

Comment:
Just a comment, no action needed: I'm a bit surprised that spelling passes with this revised list (i did check locally), given it is much shorter than the original, but I guess the updates changes settings so that some of the items in the word list are no longer necessary.

Reply:
There are two wordlists now, `.wordlist.txt` and `custom_wordlist.txt` and together they are larger than previous `.wordlist.txt`. My guess is that the `wordlist` is supposed to be a direct copy from documentation starterpack repo and `custom_wordlist` is for project-specific words.

Reply:
That is my assumption too, but the combination of the two lists still miss a lot of project specific words in the original `.wordlist.txt`, with still passing checks, so something else has changed simultaneously.